### ISSUE TITLE
Catch internal error in promise.

### ISSUE THREAD

**Issue by idelburgo** [justadudewhohacks/face-api.js#342]
Hi, 

I get this error in when calling the promise to detect face + landmarks. Is there any possibility to catch it? Im trying to get the error through then/catch but not reaching this code.

`Fragment shader compilation failed.`
and 
`Uncaught (in promise) Error: Failed to compile fragment shader.`

**Comment by justadudewhohacks** [comment]
I think being able to catch this error would require some solution similar to what is described here: https://github.com/tensorflow/tfjs/issues/756. 

Did you try to wrap the failing code into a simple try/catch? Does it not work?

**Comment by idelburgo** [comment]
Yes, I tried, but there is not error Throwed just printed and continue, so it keeps giving errors... :(
